### ISSUE TITLE
Fold/Unfold not working

### ISSUE THREAD

**Issue by summeremacs** [localauthor/org-side-tree#5]
Hi again,
When I hit <TAB> to fold/unfold in org-side-tree, I get the following error:

indent-relative: Buffer is read-only: #<buffer <tree>todo.org> [2 times]
Error running timer ‘org-side-tree-timer-function’: (error "Invalid search bound (wrong side of point)")

It doesn't fold/unfold for me. Is it supposed to on TAB or another key bind?

**Comment by localauthor** [comment]
Folding is controlled by `outline-minor-mode`, so keybindings and the behavior of outline-related commands will be determined by your own configuration of that mode. `org-side-tree` simply turns on the mode when `org-side-tree-enable-folding` is non-nil.

So, long story short, try doing this:
(setq outline-minor-mode-cycle t)

And see `outline-minor-mode` itself for any other config options you may want to fiddle with.

**Comment by summeremacs** [comment]
Thank you so much! It works and it gave me the opportunity to learn a bit more too. I set the key bind of C-c s to open org-side-tree but it ended up conflicting with another key bind I set before. In the end, I started putting all my personally added key binds at the bottom of my config so I can manage them better. Basically, it made me clean up my config a little more, so tysm. 🙃 Now I hit C-c s and org-side-tree comes up np and it works great. 👍
